src/test/java/io/debezium/connector/yugabytedb/YugabyteDBSnapshotTest.java

```diff
@@ -886,15 +959,75 @@ public void snapshotTwoColocatedNonEmptyAndNonColocatedNonEmptyThenStream(boolea
 
     @ParameterizedTest
     @ValueSource(booleans = {true, false})
-    public void verifyConnectorFailsIfMarkSnapshotDoneFails(boolean colocation) throws Exception {
+    public void snapshotTwoColocatedNonEmptyAndNonColocatedNonEmptyThenStreamWithConsistentSnapshot(boolean colocation) throws Exception {
+        /* The objective of this test is to verify that we are able to consume all snapshot records on a
```

Comment:
nit: fix comment style

Reply:
Moreover, this test only differs with the previous one in the sense that the non-colocated tables are non empty here. Is it possible to parameterise? If it makes things too complicated, leave it like this.